packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -97,7 +97,15 @@ export type GPCPCDArgs = {
    */
   id?: StringArgument;
   // TODO(POD-P3): Support PODValue of multiple types.
-};
+} /**
```

Comment:
I don't think the documentation here will be properly extracted for developer docs (see https://docs.pcd.team).  Docs need to be attached to something named, like a type or a field.
An option might be to make a named type for this Record to put detailed docs there.  The GPCArgs type docs should also explain the presence of these special fields.